packages/courDeCassation/src/scripts/updateRoute.ts

```diff
@@ -59,20 +73,3 @@ async function updateRoute(status: documentType['status']) {
     );
```

Comment:
```suggestion
    );
    if(newRoute === 'automatic'){
        documentRepository.updateStatusById(documentsToUpdate[index]._id, "done");
    }
```
Je pense que c'est mieux de mettre la logique ici :
- ça évite d'avoir de la logique dans le repository
- a cette endroit on est sur que le document est en status `free` car c'est défini en dur dans le code, le risque selon moi de le mettre dans le repository est que le `updateStatusById` soit appelé dans un autre contexte qui peut poser problème. Si un jour on appelle `updateRouteById` sur un document `loaded` et que sa nouvelle route est automatique alors il passe a done alors qu'il n'est pas annoté par le moteur